libr/util/str.c

```diff
@@ -3467,12 +3467,19 @@ R_API const char *r_str_bool(int b) {
 }
 
 R_API bool r_str_is_true(const char *s) {
-	return !r_str_casecmp ("yes", s) \
-		|| !r_str_casecmp ("on", s) \
-		|| !r_str_casecmp ("true", s) \
+	return !r_str_casecmp ("yes", s)
+		|| !r_str_casecmp ("on", s)
+		|| !r_str_casecmp ("true", s)
 		|| !r_str_casecmp ("1", s);
 }
 
+R_API bool r_str_is_false(const char *s) {
+	return !r_str_casecmp ("no", s)
+		|| !r_str_casecmp ("off", s)
+		|| !r_str_casecmp ("false", s)
+		|| !r_str_casecmp ("0", s);
+}
+
 R_API bool r_str_is_bool(const char *val) {
 	if (!r_str_casecmp (val, "true") || !r_str_casecmp (val, "false")) {
```

Comment:
why is_bool is not using is_true || is_false ? :D

Reply:
oh ok 430d0f91b76473336c24a00d48f8d28cf8854785 didn't realize that :D